.gitattributes

```diff
@@ -0,0 +1 @@
+* text=auto eol=lf
```

Comment:
This will make sure that when git clones your repo, it'll use LF on all machines for automatically detected text files.
(prettier didn't like CRLF)